pykickstart/commands/logvol.py

```diff
@@ -621,9 +622,7 @@ def parse(self, args):
         retval = F12_LogVol.parse(self, args)
         # the logvol command can't be used together with the autopart command
         # due to the hard to debug behavior their combination introduces
-        if self.handler.autopart.seen:
-            errorMsg = _("The logvol and autopart commands can't be used at the same time")
-            raise KickstartParseError(errorMsg, lineno=self.lineno)
+        self._checkConflictingCommands(_("The logvol and %s commands can't be used at the same time"))
```

Comment:
I think this can be automated as well.

Reply:
This took me a bit to understand. Do you mean that every command should run this code, so that the check can't be forgotten? That sounds good to me, it will be a noop with empty conflicts anyway.